examples/sycl/pvc/flash_attention_v2/pvc_flash_attn_epilogue.hpp

```diff
@@ -216,26 +215,31 @@ class CollectiveEpilogueAttention<
     static constexpr int FragsM = get<0>(SubgroupTileShape{}) / get<0>(MmaAtomShape()); // A frags per sub_group
     static constexpr int FragsN = get<1>(SubgroupTileShape{}) / get<1>(MmaAtomShape()); // B frags per sub_group
 
-    static constexpr int FragmentSize = (get<0>(MmaAtomShape()) * get<1>(MmaAtomShape())) / SubgroupSize;
+    static constexpr int Vec = (get<0>(MmaAtomShape()) * get<1>(MmaAtomShape())) / SubgroupSize;
 
     // Indexing variables
     auto [m_coord, n_coord, k_coord, l_coord] = tile_coord;
     auto m_offset = m_coord * BLK_M + (get_sub_group_id() / ATOM_N) * SG_M;
     auto n_offset = n_coord * BLK_N + (get_sub_group_id() % ATOM_N) * SG_N;
     auto l_offset = l_coord;
-
+    auto g = syclcompat::get_nd_item<1>().get_sub_group();
+    
     CUTLASS_PRAGMA_UNROLL
-    for (int x = 0; x < FragmentSize; x++) {
+    for (int y = 0; y < FragsM; y++) {
       CUTLASS_PRAGMA_UNROLL
-      for (int y = 0; y < FragsM; y++) {
-        
-        ElementLSE curr_sum = sum(x, y);
-        ElementO scale = (curr_sum == 0.f || curr_sum != curr_sum) ? 1.f : sycl::native::recip(curr_sum);
-        tLSEr(x, y) = curr_sum == 0.f ? -INFINITY : max(x, y) * softmax_scale + sycl::native::log(curr_sum);
-        
+      for (int x = 0; x < Vec; x++) {
+        int indx = y * Vec + x;
+        ElementLSE cur_sum {sub_group_reduce_add(sum(indx))
+                                              }; 
```

Comment:
nit: consistent formatting:
```suggestion
        ElementLSE cur_sum = sub_group_reduce_add(sum(indx)); 
```
I don't think the brace init style is used in other places.